docs/src/main/asciidoc/_admin-guide/subsystem-configuration/MicroProfile_Metrics.adoc

```diff
@@ -70,27 +71,27 @@ If the authentication fails, the  server will reply with a `401 NOT AUTHORIZED`
 The HTTP endpoint exposes the following metrics:
 
 * Base metrics - Required metrics specified in the MicroProfile 1.1 specification are exposed in the `base`  scope.
-* Vendor metrics - Metrics from WildFly subsystems are exposed in the `vendor` scope
 * Application metrics - Metrics from the application and from the deployment's subsystems are exposed in the `application` scope.
 
-=== WildFly Metrics Description
-
-WildFly metrics can appears in two scopes:
+The WildFly metrics (that measures activity in the subsystem or application deployments) are exposed only on the `/metrics`
+endpoint with the Prometheus format.
 
-* `vendor` for metrics defined in subsystem resources (e.g. the `bytes-received` attribute of the `/subsystem=undertow/server=default-server/http-listener=default` resource)
-* `applications` for metrics defined in deployment resources (e.g. the `request-count` attribute on the `/deployment=<deployment name>/subsystem=undertow/servlet=<servlet name>` resource).
+=== WildFly Metrics Description
```

Comment:
WildFly Metrics is visible when accessing http://localhost:9990/metrics
WildFly Metrics is NOT visible when accessing http://localhost:9990/metrics/
The difference is trailing `/`

Reply:
yes, the endpoint is `/metrics` (but I'll patch it so that it also works with `/metrics/`)

Reply:
fixed